pages/patterns/select-language/two-languages.md

```diff
@@ -117,7 +117,7 @@ Place the language selector button at the top of the screen above site navigatio
 ## See pattern in use
 
 {:.usa-content-list}
-- [USA.gov (beta)](http://beta.usa.gov/)
+- [USA.gov (beta)](https://www.usa.gov/)
```

Comment:
**suggestion:** Looks like we can remove the beta from this link!

```suggestion
- [USA.gov](https://www.usa.gov/)
```

Reply:
Resolved in [8d5b635](https://github.com/uswds/uswds-site/pull/2995/commits/8d5b6352260a7ea1e0ffa16488086118b68b7384)